initializr-service/src/main/resources/application.yml

```diff
@@ -398,6 +398,17 @@ initializr:
             - versionRange: "[1.2.0.RELEASE,1.3.0.RC1)"
               version: 1.0.1.RELEASE
           scope: test
+        - name: Spring Cloud Open Service Broker
```

Comment:
We don't usually prefix with "Spring Something", `Open Service Broker` sounds good enough. @philwebb suggested a `Cloud Support` group which I think is really a good idea. No need to act on it as I've something locally that does that.